src/cv2pdb.cpp

```diff
@@ -180,18 +180,18 @@ bool CV2PDB::openPDB(const TCHAR* pdbname, const TCHAR* pdbref)
 	printf("DBI::QueryImplementationVersion() = %d\n", dbi->QueryImplementationVersion());
 #endif
 
-	rc = pdb->OpenTpi("", &tpi);
+	// The default is "r" mode.  We need "rw" mode so TPI gets created if it does not exist
+	rc = pdb->OpenTpi("rw", &tpi);

```

Comment:
Sounds good. Is there official documentation for this by now?

Reply:
This might fix #87